priv/repo/migrations/20220627162154_create_items.exs

```diff
@@ -6,6 +6,7 @@ defmodule App.Repo.Migrations.CreateItems do
       add(:text, :string)
       add(:person_id, :integer)
       add(:status, :integer)
+      add(:position, :integer)
```

Comment:
We cannot alter a previous migration as it will not be re-run when we run `mix ecto.migrate` on `Fly.io` 🙃 
Going to remove this and use `list_items.position` instead. 👌